### PR TITLE
Committing to BranchX

### PR DESCRIPTION
**Pull request by maheshwariksa**
I have committed Car class. Please review.

### CODE REVIEW

File: src/Car.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+public class Car {
```
Comment (appusamysm) [inline src/Car.java:2]: this is a comment

File: src/Car.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+	
+	String name;
+	String color;
+	int number;
```
Comment (appusamysm) [inline src/Car.java:6]: Please add getters and setters

**Comment by maheshwariksa** [comment]
Hi Setter added

**Comment by appusamysm** [comment]
All look ok,pls merge.

**Pull request merged**